utils/utils.go

```diff
@@ -180,7 +180,7 @@ func CreateClient(conf NetConf) (*client.Client, error) {
 	log.Infof("Configured environment: %+v", os.Environ())
 
 	// Load the client config from the current environment.
-	clientConfig, err := client.LoadClientConfig("")
+	clientConfig, err := client.LoadClientConfig(conf.DatastoreFile)
```

Comment:
Would be nice to log out the values _after_ client config has been read in. That way users can see what values were picked up.